Lib/site.py

```diff
@@ -168,6 +169,14 @@ def addpackage(sitedir, name, known_paths):
     else:
         reset = False
     fullname = os.path.join(sitedir, name)
+    try:
+        st = os.lstat(fullname)
+    except OSError:
```

Comment:
Hum, maybe add a _trace() call to help debugging such issue. Something like:

```
_trace(f"Skipping .pth file, stat failed: {exc!r}")
```

Reply:
When `open()` fails below, it is simply silently returns.

If we want to add a `_trace()` for OSError, we should do this in both cases (and maybe in more cases). But this is a different issue. It can add a noise where it was silent.

Reply:
Most `OSError`s are silently ignored in this module and do not have a comment. I do not know what to add in these two places and why they are special. I also not sure that it is a deliberate choice, it can just be a copied behavior.

Reply:
As you want, I already approved the PR.